index.html

```diff
@@ -4,12 +4,18 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <script src="project1.js"></script>
+    <title>Tades to-do</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" />
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <script>
-       alert(`Answer: ${no1} ${symbol} ${no2} = ${solution}`);
-    </script>
+    <div class="app">
+        <h1>TO-DO LIST</h1>
+       <input placeholder="input task here...." id="input"></i><span id="btn1"><i class="fas fa-plus"></i>Add</span>
```

Comment:
Please rename the id from 'btn1' to something else that is more descriptive of what the button does. It doesn't change functionality but is important to build the right habits early